airbyte-integrations/connectors/source-chargify/source_chargify/schemas/subscriptions.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "properties": {
+    "id": {
+      "type": ["integer"]
+    },
+    "state": {
+      "type": ["string"]
+    },
+    "balance_in_cents": {
+      "type": ["integer"]
+    },
+    "total_revenue_in_cents": {
+      "type": ["integer"]
+    },
+    "product_price_in_cents": {
+      "type": ["integer"]
+    },
+    "product_version_number": {
+      "type": ["integer"]
+    },
+    "current_period_ends_at": {
+      "type": ["string"]
+    },
+    "next_assessment_at": {
+      "type": ["string"],
+      "format": "date-time"
+    },
+    "trial_started_at": {
+      "type": ["string"],
+      "format": "date-time"
+    },
+    "trial_ended_at": {
+      "type": ["string"],
+      "format": "date-time"
+    },
+    "activated_at": {
+      "type": ["string"],
+      "format": "date-time"
+    },
+    "expires_at": {
+      "type": ["string"],
+      "format": "date-time"
+    },
+    "created_at": {
+      "type": ["string"],
+      "format": "date-time"
+    },
+    "updated_at": {
+      "type": ["string"],
+      "format": "date-time"
+    },
+    "cancellation_message": {
+      "type": ["string"]
```

Comment:
Are you sure all the fields you declared below are really not nullable?

Reply:
schemas are recreated!